packages/core/strapi/lib/services/cron.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+const cron = require('node-schedule');
+const { isFunction, getOr } = require('lodash/fp');
+
+const createCronService = () => {
+  const groupsOfTasks = [];
+  let jobs = [];
+
+  return {
+    add(newTasks = {}) {
+      if (!strapi.config.get('server.cron.enabled')) {
+        return;
+      }
+
+      groupsOfTasks.push(newTasks);
+    },
+    start() {
+      for (const tasks of groupsOfTasks) {
+        for (const taskExpression in tasks) {
+          const taskValue = tasks[taskExpression];
+
+          if (isFunction(taskValue)) {
+            const instanciatedTask = taskValue(strapi);
```

Comment:
I think the function should be passed directly there are no  instantiation phase ?